src/mlpack/core/dists/laplace_distribution.hpp

```diff
@@ -158,6 +179,17 @@ class LaplaceDistribution
   double scale;
 };
 
+inline void LaplaceDistribution::LogProbability(
+    const arma::mat& x,
+    arma::vec& logProbabilities) const
+{
```

Comment:
I guess, we could also put this inside the class itself, this is what was done for the ``Random`` method.

Reply:
I agree, that would be an improvement.  I'll do it during merge.